static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -198,7 +210,7 @@ const canShow = async (): Promise<boolean> => {
     if (!canShowPreChecks({
         brazeSwitch,
         apiKey,
-        isDigiSubscriber: isDigitalSubscriber(),
+        userIsGuSupporter: shouldNotBeShownSupportMessaging(),
```

Comment:
I think this line could use a comment to explain why this function is aligned with our goals for the behaviour here. There's an assumption that the team maintaining this function will keep it working the way we want and that new products don't require changes that extend beyond this logic. I am sure going to cause a bug in the future. A description of our intent here might help a developer fix the bug when that inevitably happens down the road.

Reply:
Good idea, it is quite unintuitive. I've added a comment now along the lines of the one on DCR.